scripts/override_respot.py

```diff
@@ -102,24 +102,7 @@ def override_values(tech, year, dr):
         else:
             print("No RES potential techs to override...")
 
-        if (
```

Comment:
@Eddy-JV why is this deleted? Indeed, it does not look like nice code... But what did it do and do we still need it?

Reply:
I deleted it because it looks like it is only hard coded for Marocco. Which will lead to erros if another country is used if both  
 snakemake.config["custom_data"]["add_existing"]
 and snakemake.wildcards["planning_horizons"] == "2030"

are satisfied.